rs/boundary_node/rate_limits/canister/access_control.rs

```diff
@@ -0,0 +1,54 @@
+use candid::Principal;
+use mockall::automock;
+
+use crate::storage::API_BOUNDARY_NODE_PRINCIPALS;
+
+const FULL_ACCESS_ID: &str = "imx2d-dctwe-ircfz-emzus-bihdn-aoyzy-lkkdi-vi5vw-npnik-noxiy-mae";
```

Comment:
This should also be removed and set through the init/upgrade args. Maybe we can add two vecs to those args: one vec with the principals that should be added and one vec with the principals that should be removed. Also, we might want to add a method that exposes the principals with full access.

Reply:
> This should also be removed and set through the init/upgrade args. 

I can do it, but why? Is this because You wanted this principal to be eventually retrieved via canister call?